rest/data/project_settings.go

```diff
@@ -237,6 +237,17 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 			}
 		}
 
+		if mergedSection.Owner != "" && mergedSection.Repo != "" {
```

Comment:
Is this if block in place to avoid erroring if we called `ValidateOwnerAndRepo` directly? Can we get around this by exporting the `validateOwner` function, and then just calling that here, since it only checks the list of valid orgs and won't error if the Owner or Repo is empty?